rust/src/ssh/logger.rs

```diff
@@ -16,49 +16,40 @@
  */
 
 use super::ssh::SSHTransaction;
-use crate::json::*;
+use crate::jsonbuilder::{JsonBuilder, JsonError};
 
-fn log_ssh(tx: &SSHTransaction) -> Option<Json> {
+fn log_ssh(tx: &SSHTransaction) -> Result<Option<JsonBuilder>, JsonError> {
```

Comment:
this should not return a JsonBuilder, but take a JsonBuilder as input that it will add its output to.

Reply:
ok, but this induces adding empty json for some alerts (see next PR)